### PR TITLE
Do not save empty PointSet, add additional null checks to mitkSceneIO

### PR DESCRIPTION
**Pull request by mpkh**

### CODE REVIEW

File: Modules/SceneSerialization/src/mitkPointSetSerializer.cpp [hunk 1]
```diff
@@ -37,6 +37,12 @@ std::string mitk::PointSetSerializer::Serialize()
     return "";
   }
 
+  // do not save empty PointSet
+  if (ps->GetSize() == 0)
```
Comment (maleike) [inline Modules/SceneSerialization/src/mitkPointSetSerializer.cpp]: Please allow me adding my thoughts on this one. With this change applied, do we still get the DataNode around this PointSet correctly restored? This could be important conserve the hierarchy of DataStorage unmodified. I would discourage skipping certain data objects due to considerations about their usefulness (their properties or their node's properties could still be significant for the application).

Comment (mpkh) [reply Modules/SceneSerialization/src/mitkPointSetSerializer.cpp]: Agreed, I actually need to re-test if problems with saving empty point set are still being actual or not, will close this PR for now
